stripe/src/main/java/com/stripe/android/model/SourceTypeModel.kt

```diff
@@ -9,33 +8,30 @@ import kotlinx.android.parcel.Parcelize
 sealed class SourceTypeModel : StripeModel {
     @Parcelize
     data class Card internal constructor(
-        val addressLine1Check: String?,
-        val addressZipCheck: String?,
+        val addressLine1Check: String? = null,
```

Comment:
should the params w/ default values be last in the order? (i.e. put brand first)

Reply:
there isn't an overloaded version of the constructor so it shouldn't make a difference